### ISSUE TITLE
Ticket List not displaying correctly

### ISSUE THREAD

**Issue by wesbillman** [zendesk/zendesk_sdk_ios#194]
It seems to be broken in both the sample app and when I try to use the latest version of the sdk in my app.

Version 1.6.2.1: 8fd049cfebed51bdcb2bb5d21f0599f6aad9af83

Here's what it looks like while it's transitioning from "Ticket List" button tapped to "My Tickets":
![simulator screen shot jun 26 2016 11 13 01 am](https://cloud.githubusercontent.com/assets/51647/16363948/0825cb40-3b8f-11e6-9683-48ab3b848e0c.png)

Here's what it looks like once "My Tickets" is show:
![simulator screen shot jun 26 2016 11 06 58 am](https://cloud.githubusercontent.com/assets/51647/16363929/9c067fcc-3b8e-11e6-9fdd-67f4b64648da.png)

Here's what it looks like in my app:
![simulator screen shot jun 26 2016 11 09 23 am](https://cloud.githubusercontent.com/assets/51647/16363932/a4cd1e22-3b8e-11e6-8229-b562a0543649.png)

**Comment by mathewcropper** [comment]
Hi Wes. Thanks for reporting this. I just tried to replicate it and wasn't able to do so. I was using a build target of iOS 7.0, and ran the sample app in simulator for both an iPhone 6S and an iPhone 6S Plus. I'm also using Xcode 7.3, and the simulator version is 9.3.

Would you mind sharing some more about your setup, please? It'd be ideal if you were able to capture logs as well (instructions here).

**Comment by wesbillman** [comment]
Hey @mathewcropper I literally just downloaded the v1.6.2.1 release and unzipped it. Then selected iPhone 6s Plus (9.3) and hit ran. After entering my url, id, client id and tapping done, I tapped on "Ticket List". And I can see the issue. I actually see it on older releases too. v1.6.0.1 and v1.6.1.1.

```
2016-06-26 11:53:33.650 SDK-Sample[4434:328487] [DEBUG]: Reachability Flag Status: -R ------- networkStatusForFlags
2016-06-26 11:54:34.329 SDK-Sample[4434:328487] configuration made, url:REDACTED
2016-06-26 11:54:43.129 SDK-Sample[4434:328487] [INFO]: Localization key and value : "ios.ZDKRequests.requestList.title" = "My tickets";
2016-06-26 11:54:43.144 SDK-Sample[4434:328487] [INFO]: Localization key and value : "ios.ZDKRequests.rightBarButton.title.new" = "New";
2016-06-26 11:54:43.145 SDK-Sample[4434:328487] [DEBUG]: SDK configuration file found in local storage
2016-06-26 11:54:43.146 SDK-Sample[4434:328487] [DEBUG]: SDK configuration file found in local storage
2016-06-26 11:54:43.146 SDK-Sample[4434:328487] [DEBUG]: Reachability Flag Status: -R ------- networkStatusForFlags
2016-06-26 11:54:43.146 SDK-Sample[4434:328487] [WARNING]: Conversations disabled, this feature is not available on your plan or was disabled.
2016-06-26 11:54:43.146 SDK-Sample[4434:328487] [INFO]: Localization key and value : "ios.ZDKRequests.requestList.error" = "Failed to get tickets";
2016-06-26 11:54:43.146 SDK-Sample[4434:328487] [DEBUG]: Reachability Flag Status: -R ------- networkStatusForFlags
2016-06-26 11:54:43.147 SDK-Sample[4434:328487] [INFO]: Localization key and value : "ios.ZDKRequests.requestList.error" = "Failed to get tickets";
2016-06-26 11:54:43.147 SDK-Sample[4434:328487] [INFO]: Localization key and value : "ios.ZDKHelpCenter.toastRetryButton" = "Retry";
2016-06-26 11:54:43.149 SDK-Sample[4434:328487] [DEBUG]: Failed to get request list: displaying toast
2016-06-26 11:54:43.661 SDK-Sample[4434:328487] [DEBUG]: Reachability Flag Status: -R ------- networkStatusForFlags
```

I'm using Xcode: Version 7.3.1 (7D1014).

You can try slowing down animations to see that the transition after clicking "Ticket List" is animating the view in as it transitions. It looks like there are some autolayout issues when this view is show.

**Comment by mathewcropper** [comment]
Thanks, Wes. That was exactly what I needed. I've found the following:

* Your account has conversations disabled. When that's the case, when you access the request list a toast is displayed, with an error.
* When running in simulator at normal speed, I'm not able to recreate this.
* With slow animations enabled, I get the same thing, namely that the toast never properly appears.
* After running it with slow animations, when returning to normal speed, the problem persists.

I've added a gif to this comment to show the issue.

Thanks for flagging this. We'll take a look and come back to you, Wes.

![2016-06-26 20 17 49](https://cloud.githubusercontent.com/assets/5889485/16364287/1d099c7a-3bdb-11e6-9c3f-7c78d70430a3.gif)
